### PR TITLE
[DB]: model creation and user migration

### PR DESCRIPTION
**Pull request by HugoCavalero92**
## Jira Ticket WA-1

- [WA-1]

## Description
Model creation and user migration
AS: Administrator
I Want: To have control
To: Of users

## Evidence:
DBUser

### CODE REVIEW

**Review by germanWalton — APPROVED** [review]

File: database/migrations/20221102141110-create-user.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        type: Sequelize.STRING
+      },
+      password: {
+        type: Sequelize.STRING
+      },
+      avatar: {
+        type: Sequelize.STRING
+      },
+      roleId: {
+        type: Sequelize.INTEGER
```
Comment (ledelaespriella) [inline database/migrations/20221102141110-create-user.js]: ### Pending

add the foreign key in the migration:

` roleId: {
        type: Sequelize.INTEGER,
references: {
          model: {
            tableName: 'Role',
            schema: 'schema'
          },
          key: 'id'
        },
        allowNull: false
},`

**Review by ledelaespriella — CHANGES_REQUESTED** [review]
Please correct migrations

File: database/migrations/20221102141110-create-user.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+      },
+      password: {
+        type: Sequelize.STRING
+      },
+      avatar: {
+        type: Sequelize.STRING
+      },
+      roleId: { 
+        type: Sequelize.INTEGER, 
+        references: { model: { tableName: 'Role', schema: 'schema' }, key: 'id' },
```
Comment (ledelaespriella) [inline database/migrations/20221102141110-create-user.js]: ### Pendingn

Code correct:

`roleId: { 
        type: Sequelize.INTEGER, 
        references: { 
          model: 'Roles' , 
          key: 'id' 
        },
        allowNull: false 
      },`

View in mysql Workbench:

Captura de Pantalla 2022-11-03 a la(s) 2 39 37 p m

**Review by ledelaespriella — CHANGES_REQUESTED** [review]
Please correct migrations

**Review by ledelaespriella — APPROVED** [review]
### **passed**

**Pull request merged**